apps/docs/content/components/Section/index.mdx

```diff
@@ -1,151 +1,51 @@
-| ------------------------- | ----------------------------------------------------------------------------------------------------- | ---------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `paddingBlockStart`       | <PropTableValues values={[...SectionPaddingVariants]} commaSeparated  /> `ResponsiveMap`              | `'normal'` | `false`  | Control the padding at the top of the section.                                                                                                                |
-| `paddingBlockEnd`         | <PropTableValues values={[...SectionPaddingVariants]} commaSeparated  /> `ResponsiveMap`              | `'normal'` | `false`  | Control the padding at the bottom of the section.                                                                                                             |
-| `backgroundColor`         | <PropTableValues values={[...SectionBackgroundColors]} commaSeparated  /> \| `string` `ResponsiveMap` |            | `false`  | Add a background color. Use a predefined color, a CSS custom property, or a custom color value.                                                               |
-| `backgroundImageSrc`      | `string` \| `string[]` `ResponsiveMap`                                                                |            | `false`  | Add one or more background images.                                                                                                                            |
-| `backgroundImageSize`     | `string` \| `string[]` `ResponsiveMap`                                                                | `'cover'`  | `false`  | Control the size of the background image(s). This accepts all CSS `background-size` supported values, including keywords like `contain`.                      |
-| `backgroundImagePosition` | `string` \| `string[]` `ResponsiveMap`                                                                | `'50%`     | `false`  | Control the position of the background image(s). This accepts all CSS `background-position` supported values, including keywords like `top` and edge offsets. |
+import ComponentLayout from '../../../src/layouts/component-layout'
+import anatomy from './images/anatomy.png'
+
+export default ComponentLayout
+
+## Anatomy
+
+<img
+  src={anatomy}
+  alt="The anatomy of the section component illustrates the various elements that make it up."
+/>
+
+
+
+- **Container:** Wraps the content to ensure proper alignment and responsiveness across different viewports.
+- **Padding:** Customizable padding at the top and bottom of the section to suit different content needs.
+- **Background:** Options to customize the background with colors or images.
+
+## Usage
+
+The Section component is a versatile layout tool designed  It serves as a container that can be customized to fit various design needs.
```

Comment:
Looks like this might be two sentences mixed together 🙂

Reply:
Indeed, that was a poor copy and paste. Thank you!